crates/dekaf/src/log_appender.rs

```diff
@@ -383,14 +386,24 @@ impl<W: TaskWriter + Clone + 'static> TaskForwarder<W> {
                 // Poll log and stats appends, if running
                 _ = &mut log_append_state, if !log_append_state.is_terminated() => {
                     if let Some(Err(e)) = Pin::new(&mut log_append_state).take_output(){
-                        tracing::error!(error = ?e, "Error appending logs");
+                        tracing::error!(error = ?e, ?backoff_duration, "Error appending logs");
+
+                        tokio::time::sleep(backoff_duration).await;
+                        backoff_duration = (backoff_duration * 2).min(max_backoff);
```

Comment:
It doesn't seem like anything ever resets the `backoff_duration`, and I'm wondering whether that could pose a problem. If this loop is potentially very long lived, then I'd probably want to reset the backoff duration after a successful append. Or maybe just use a fixed backoff for now.

Reply:
Ah, nvm, I didn't notice the reset below. Disregard.